tests/lib/snaps/store/test-snapd-content-plug-to-classic-slot/meta/snap.yaml

```diff
@@ -0,0 +1,12 @@
+name: test-snapd-content-plug-to-classic-slot
```

Comment:
The snaps in store directory are the ones which are created through snapcraft.yaml, this should go in the test directory and be installed doing:
"$TESTSTOOLS"/snaps-state install-local test-snapd-content-plug-to-classic-slot